deep-learning/_posts/2019-04-04-BreastCancerScreening.md

```diff
@@ -0,0 +1,53 @@
+---
+layout: review
+title:  "Deep Neural Networks Improve Radiologists' Performance in Breast Cancer Screening"
+tags:   deep-learning medical-imaging
+author: Antoine Théberge
+pdf:    "https://arxiv.org/abs/1903.08297"
+cite:
+    authors: "Nan Wu, Jason Phang, Jungkyu Park, Yiqiu Shen, Zhe Huang, Masha Zorin, Stanisław Jastrzębski, Thibault Févry, Joe Katsnelson, Eric Kim, Stacey Wolfson, Ujas Parikh, Sushma Gaddam, Leng Leng Young Lin, Kara Ho, Joshua D. Weinstein, Beatriu Reig, Yiming Gao, Hildegard Toth, Kristine Pysarenko, Alana Lewin, Jiyon Lee, Krystal Airola, Eralda Mema, Stephanie Chung, Esther Hwang, Naziya Samreen, S. Gene Kim, Laura Heacock, Linda Moy, Kyunghyun Cho, Krzysztof J. Geras"
+    title:   "Deep Neural Networks Improve Radiologists' Performance in Breast Cancer Screening"
+    venue:   "CVPR 2019"
+---
+
+[Blog post](https://medium.com/@jasonphang/deep-neural-networks-improve-radiologists-performance-in-breast-cancer-screening-565eb2bd3c9f)
+
+[Code and trained model](https://github.com/nyukat/breast_cancer_classifier)
+
+[Technical report](https://cs.nyu.edu/~kgeras/reports/datav1.0.pdf)
+
+## Introduction
+
+_"Breast cancer is the second leading cancer-related cause of death among women in the US. Early detection, through routine annual screening mammography, is the best first line of defense against breast cancer. [...] A radiologist can spend up to 10 hours a day working through these mammograms, in the process experiencing both eye-strain and mental fatigue."_
+
+The authors propose a deep-learning method leveraging both pixel-level and patch-level precision that is as accurate as ensembles of radiologists. They also provide a new dataset that was used to train the proposed model.
+
+## Dataset
+
+Two challenges arise in breast cancer detection. One, the most commonly used mammography dataset contains only about 10 000 images. Second, mammography images are very high resolution (around 2000x2000). [Prior work](https://arxiv.org/abs/1703.07047) shows that downsampling these images can significantly hurt performance of computer vision model, so it is not a viable option.
```

Comment:
_(...) performance of computer vision model, (....)_ -> _(...) performance of computer vision models, (....)_